iceoryx_posh/test/integrationtests/service_discovery/test_roudi_findservice.cpp

```diff
@@ -40,6 +40,70 @@ TEST_F(RoudiFindService_test, OfferSingleMethodServiceSingleInstance)
     ASSERT_THAT(*instanceContainer.begin(), Eq(IdString("instance1")));
 }
 
+TEST_F(RoudiFindService_test, OfferServiceWithDefaultDescription)
```

Comment:
Could you please adjust your unit test names so that they reflect what you are testing and what you are expecting. Something like: OfferedServiceWithValidServiceDescriptionCanBeFound

Reply:
done